src/containers/guest-home-page/WhatCanYouDo.tsx

```diff
@@ -30,10 +30,21 @@ const cardData = [
 
 const WhatCanYouDo = () => {
   const { t } = useTranslation()
+  const { openModal } = useModalContext()
+  const [selectedRole, setSelectedRole] = useState<UserRoleEnum | null>(null)
```

Comment:
Можна по ідеї передати дефолте значення ролі при визначенні стейту: 
```suggestion
const [selectedRole, setSelectedRole] = useState<UserRoleEnum>(UserRoleEnum.Student)
```

Reply:
Так, дійсно. Чим менше рядкових літералів, тим краще